atlas/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+# There is no pre-build tarball available if not using Dataproc 1.5+
+# Following commands can be used to build Atlas 1.2.0 from source
+
+# install maven 3.6
+wget https://archive.apache.org/dist/maven/maven-3/3.6.0/binaries/apache-maven-3.6.0-bin.tar.gz -P /opt
```

Comment:
Dataproc has Atlas in APT repo, may you use it instead: `apt-get install atlas`?

Reply:
The thoughts were to use this to allow customer build. But since let's only enable this for 1.5. Removed this build script.